### PR TITLE
fix: deploy-task.sh script

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
fix: deploy-task.sh script

deploy-task.sh script incorrectly replaces URL to images when used in local envs.

**Release note**:
```
NONE
```

### CODE REVIEW

File: scripts/deploy-tasks.sh [hunk 1]
```diff
@@ -34,12 +35,12 @@ visit "${REPO_DIR}/release/tasks"
...
         kubectl apply -n ${DEPLOY_NAMESPACE} -f "${TASK_NAME}.yaml"
       else
-        TASKS_IMAGE="quay.io/kubevirt/tekton-tasks:${RELEASE_VERSION}"
-        DISK_VIRT_IMAGE="quay.io/kubevirt/tekton-tasks-disk-virt:${RELEASE_VERSION}"
-
-        sed "s!${TASKS_IMAGE}!${TEKTON_TASKS_IMAGE}!g" "${TASK_NAME}.yaml" | \
-        sed "s!${DISK_VIRT_IMAGE}!${TEKTON_TASKS_DISK_VIRT_IMAGE}!g" "${TASK_NAME}.yaml"  | \
-        kubectl apply -n ${DEPLOY_NAMESPACE} -f -
+        TASKS_IMAGE="quay.io/kubevirt/tekton-tasks"
+        DISK_VIRT_IMAGE="quay.io/kubevirt/tekton-tasks-disk-virt"
```
Comment (jcanocan) [inline scripts/deploy-tasks.sh:39]: Why does the `RELEASE_VERSION` have to be dropped?

Comment (ksimon1) [reply scripts/deploy-tasks.sh:39]: because the script should replace the URL no matter the release version

Comment (jcanocan) [reply scripts/deploy-tasks.sh:39]: Got it! Thanks!

**Comment by jcanocan** [comment]
Nice job!
/lgtm

**Pull request merged**